packages/cozy-konnector-libs/docs/api.md

```diff
@@ -231,6 +231,58 @@ use this function.</p>
 is already saved in the cozy</li>
 </ul>
 </dd>
+<dt><a href="#module_filterData">filterData</a></dt>
+<dd><p>This module proposes some small utils regarding connectors</p>
+<p>Parameters:</p>
+<ul>
+<li><code>documents</code>: an array of objects corresponding to the data you want to save in the cozy</li>
+<li><code>doctype</code> (string): the doctype where you want to save data (ex: &#39;io.cozy.bills&#39;)</li>
+<li><code>options</code> :<ul>
+<li><code>keys</code> (array) : List of keys used to check that two items are the same. By default it is set to `[&#39;id&#39;]&#39;.</li>
+<li><code>index</code> (optionnal) : Return value returned by <code>cozy.data.defineIndex</code>, the default will correspond to all documents of the selected doctype.</li>
+<li><code>selector</code> (optionnal object) : Mango request to get records. Default is built from the keys <code>{selector: {_id: {&quot;$gt&quot;: null}}}</code> to get all the records.</li>
+</ul>
+</li>
+</ul>
+<pre><code class="lang-javascript">const documents = [
+  {
+    name: &#39;toto&#39;,
+    height: 1.8
+  },
+  {
+    name: &#39;titi&#39;,
+    height: 1.7
+  }
+]
+
+return filterData(documents, &#39;io.cozy.height&#39;, {
+  keys: [&#39;name&#39;]
+}).then(filteredDocuments =&gt; addData(filteredDocuments, &#39;io.cozy.height&#39;))
+
+</code></pre>
+</dd>
+<dt><a href="#module_utils">utils</a></dt>
+<dd><p>This function allows to fetch all documents for a given doctype. It is the fastest to get all
+documents but without filtering possibilities</p>
+<p>Parameters:</p>
+<ul>
+<li><code>doctype</code> (string): the doctype from which you want to fetch the data</li>
+</ul>
+</dd>
+<dt><a href="#module_utils">utils</a></dt>
+<dd><p>This function allows to fetch all documents for a given doctype exceeding the 100 limit.
+It is slower that fetchAll because it fetches the data 100 by 100 but allows to filter the data
+with a selector and an index</p>
+<p>Parameters:</p>
+<ul>
+<li><code>doctype</code> (string): the doctype from which you want to fetch the data</li>
+<li><code>selector</code> (object): the mango query selector</li>
+<li><code>index</code> (object): (optional) the query selector index. If not defined, the function will
+create it&#39;s own index with the keys specified in the selector</li>
+</ul>
+<pre><code class="lang-javascript">const documents = await queryAll(&#39;io.cozy.bills&#39;, {vendor: &#39;Direct Energie&#39;})
+</code></pre>
+</dd>
```

Comment:
What the point to write html code in a markdown file?

Reply:
This is generated code actually : https://github.com/konnectors/libs/blob/3614911cc293510cd8ae29cb01b42bc789bc2ea3/packages/cozy-konnector-libs/package.json#L44